websites/D/dod Games/presence.ts

```diff
@@ -12,225 +12,237 @@ presence.on("UpdateData", async () => {
 		playerName = document.querySelector(
 			"#component_top_right_loginasname_text"
 		).textContent,
-		urls = document.querySelectorAll("#topBarDownLink");
+		urls = document.querySelectorAll("#topBarDownLink"),
+		supportedGames: string[] = ["drawit", "quiz", "agonia", "tichu"];
 	let openRooms = 0;
 
-	for (const url of urls) {
-		if (url.textContent.includes("&room=")) openRooms++;
+	function addJoinRoomButton(roomURL: string) {
+		presenceData.buttons = [
+			{
+				label: "Είσοδος",
+				url: roomURL,
+			},
+		];
+	}
+
+	function parseGameData(game: string, roomURL?: string) {
+		if (!isInGame) {
+			timeStarted = Date.now();
```

Comment:
```suggestion
			timeStarted = Math.floor(Date.now() / 1000);
```

Reply:
Discord accepts either

Reply:
ah ic ok

Reply:
I can even tell you that usually when you code a bot, Discord gives you the timestamp without the /1000 and this can be annoying sometimes.